src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -108,12 +108,12 @@ class TcgStatics {
   }
 
   //For newer formats "may" should not be used, but for older (Wizards era) ones doing a coin flip was the offered alternative if a player didn't know how to play RPS.
-  static rockPaperScissors(Closure winEff, Closure lossEff = {}, boolean may = false) {
+  static rockPaperScissors(Closure winEff, Closure lossEff = {}, multi=[:]) {
     def winnerDetermined = false
     def myWin = false
     def confirmMsg = "Do you want to play Rock - Paper - Scissors? (Either player saying no will result in a coinflip.)"
 
-    if (!may || confirm(confirmMsg) && oppConfirm(confirmMsg)) {
+    if (!params.may || confirm(confirmMsg) && oppConfirm(confirmMsg)) {
```

Comment:
Is `params.may` still accessible?

Reply:
Yeah, awfully written I'm redoing it (that's why I drafted it, haha)

Reply:
Good luck :)